misk-aws/src/test/kotlin/misk/jobqueue/sqs/SqsJobQueueTest.kt

```diff
@@ -414,7 +414,7 @@ internal class SqsJobQueueTest {
       sqsMetrics.jobsDeadLettered.labels(queueName.value, queueName.value).get()
     ).isEqualTo(0.0)
     assertThat(sqsMetrics.sqsDeleteTime.count(queueName.value, queueName.value)).isEqualTo(1)
-    assertThat(sqsMetrics.queueProcessingLag.count(queueName.value, queueName.value)).isEqualTo(3)
+    assertThat(sqsMetrics.queueProcessingLag.count(queueName.value, queueName.value)).isEqualTo(1)
```

Comment:
Is this expected? Why is this not 3 anymore?

Reply:
See `2` [above](https://github.com/cashapp/misk/pull/2893#issuecomment-1662872500), I think the previous emulator was buggy and not wasn't updating `ApproximateReceiveCount`

Reply:
Ah it's approx, thanks for clarifying.